railties/lib/rails/application/default_middleware_stack.rb

```diff
@@ -43,6 +43,7 @@ def build_stack
 
           middleware.use ::Rack::Runtime
           middleware.use ::Rack::MethodOverride unless config.api_only
+          middleware.use ::ActionDispatch::CurrentRackEnv
```

Comment:
I'm not sure we need this for every application. Right now we need because we exposed the `Renderer.current` as public API but is there any reason why people would call `Renderer.current` in their apps given they can call `ApplicationController.renderer`?